pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-apiserver
-    version: v1.3.0-beta.2
```

Comment:
I'd prefer to have label metadata, as it makes selecting on this data much easier than having to filter from the image itself.

It's also possible that the version of this "application" (as pod or manifest ) becomes separated from the image version. Or what about situations of multiple containers (and images) per pod. Then we have to filter each by name then by image (rather than by an object-level versioning).